### PR TITLE
Added sophomore page

### PR DESCRIPTION
**Pull request by jay-deep**
Bro I've added sophomore
at 25%
at 100%

 page as u assigned, but there's some one issue in code, when u zoom out ur screen to 25% Then the page will get displayed perfectly, I am working on this issue but till then Check the code and and tell me what should I update
issue no #3

### CODE REVIEW

**Comment by decon-harsh** [comment]
Checking this out ! No need to add instagram icon that was just a refrence no prob i will edit it!

**Comment by decon-harsh** [comment]
There is padding of 4000px and 700 px

File: sophmore-page/sophomore.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+h1{ font-size: 200px;
+ padding-left: 200px;
+ margin-top: 309px;
+ color: grey;
+ }
+
+.firstbutton button{
+	color: white;
+	background-color: #ff7675;
+	width: 2800px;
```
Comment (decon-harsh) [inline sophmore-page/sophomore.html:59]: 2800 px are way too much

**Comment by decon-harsh** [comment]
Sorry bro but you need practice , weak judge of margins

**Comment by decon-harsh** [comment]
Cannot merge this, this needs more work, Nice attempt @jay-deep . Do reduce the margins, paddings, take help from index.html, freshers.html, and css files.

**Comment by jay-deep** [comment]
yes bro , will send u again after correcting it

**Comment by decon-harsh** [comment]
> yes bro , will send u again after correcting it

If you commit changes in your fork , it will automatically change here.

**Comment by jay-deep** [comment]
I am writing css again , in my vs code , then will change in my fork

**Comment by decon-harsh** [comment]
Sorry i was busy didn't see the new commits. My fault

I will check it asap

Next time if you make new commit please drop a message below it , easy for me to get notifications.

**Comment by jay-deep** [comment]
ohh , okok no problem, and bro can u add "hactobefest" label in ur topic?

**Comment by decon-harsh** [comment]
> ohh , okok no problem, and bro can u add "hactobefest" label in ur topic? 

I guess it's already added doesn't it? I will add the issue labels too.

**Comment by jay-deep** [comment]
This is the new rule of hactoberfest
"Your pull requests will count toward your participation if they are in a repository with the hacktoberfest topic and once they have been merged, approved by a maintainer or labelled as hacktoberfest-accepted."
I didn't get it, can u explain me a little bit?

**Comment by decon-harsh** [comment]
Can you go to hacktober website and then login to your profile . Is this pr in maturity period?

**Comment by jay-deep** [comment]
Yes , 10days
See, I don't understand this
Screenshot (24)
And these are the new rules of hacktober fest
rules

**Comment by decon-harsh** [comment]
These rules are for PR after 3rd oct 00:00,

The time period i.e maturity period is the the time for us maintainers to check whether a PR is useful or not . Since your last PR no 20 cannot be merged and you opened a new PR 21 i have to close the former . That won't be counted for the hacktober fest.

**Pull request closed without merging**